modules/upgrade/pages/upgrade-overview.adoc

```diff
@@ -18,20 +18,21 @@ ifeval::[{suma-content} == true]
 {productname} uses an [literal]``X.Y.Z`` versioning schema.
 To determine which upgrade procedure you need, look at which part of the version number is changing.
 
-X upgrades::
+Major Version Upgrade (X Upgrade)::
 Upgrading to the next major version.
-For example, upgrading from 3.2 to 4.0.
+For example, upgrading from 3.2 to 4.0 or to 4.1.
 
-Y upgrades::
+Minor Version Upgrade (Y Upgrade)::
 Upgrading to the next minor version.
-This is often referred to as a service pack migration.
+This is often referred to as a service pack migration or SP migration.
 For example, upgrading from 4.0 to 4.1.
 
-Z upgrades::
+Patch Level Upgrade (Z Upgrade)::
 Upgrading within the same minor version.
 This is often referred to as a maintenance update.
-For example, upgrading from 4.0.0 to 4.0.2.
+For example, upgrading from 4.0.0 to 4.0.2 or from 4.1.0 to 4.1.1.
 
+// Which term do we want to use here and later?  upgrading or updating?
```

Comment:
It's an upgrade guide, let's just pick upgrade and stick to it.

Reply:
ok, it's in.